tests/api-watch-diff/test.ml

```diff
@@ -2,3 +2,214 @@ let%expect_test "test_diff_interface" =
   let result = Api_watch_diff.diff_interface ~reference:[] ~current:[] in
   Format.printf "%b" result;
   [%expect {|false|}]
+
+open Types
+
+(* Signature for type t:
+     > type t = int *)
+
+let t_sig =
+  Sig_type
+    ( Ident.create_persistent "t",
+      {
+        type_params = [];
+        type_arity = 0;
+        type_kind = Type_abstract;
+        type_private = Public;
+        type_manifest =
+          Some
+            (Transient_expr.type_expr
+               (Transient_expr.create
+                  (Tconstr
+                     (Path.Pident (Ident.create_predef "int"), [], ref Mnil))
+                  ~level:0 ~scope:0 ~id:0));
+        type_variance = [];
+        type_separability = [];
+        type_is_newtype = false;
+        type_expansion_scope = 0;
+        type_loc = Location.none;
+        type_attributes = [];
+        type_immediate = Unknown;
+        type_unboxed_default = false;
+        type_uid = Uid.internal_not_actually_unique;
+      },
+      Trec_not,
+      Exported )
+
+(* Signature for type t:
+     > type unused_type = string *)
+
+let unused_type_sig =
+  Sig_type
+    ( Ident.create_persistent "unused_type",
+      {
+        type_params = [];
+        type_arity = 0;
+        type_kind = Type_abstract;
+        type_private = Public;
+        type_manifest =
+          Some
+            (Transient_expr.type_expr
+               (Transient_expr.create
+                  (Tconstr
+                     (Path.Pident (Ident.create_predef "string"), [], ref Mnil))
```

Comment:
I think you can directly use `Predef.path_string` here. Same apply to `int` or any predefined ocaml type.